### DISCUSSION TITLE
Git LFS on existing repo: post-git add "git lfs ls-files" returns nothing

### DISCUSSION THREAD

**Discussion by gregorhd** [git-lfs/git-lfs#4572] [Q&A]
Hi,

I am trying to LFS track large files in an existing repo without success. Here is my process thus far (following [this ](https://stackoverflow.com/a/35518689/14801819)SO tip).

1. `git lfs install`
```
Updated git hooks.
Git LFS initialized.
```
2. `git lfs track "*.html"`, `git lfs track "*.svg"` and, for good measure, `git lfs track "myfolder/"` which contains the `.html `and `.svg` files I want to track.
3. add, commit and push .gitattributes
4. `.gitattributes` now reads
```
'myfolder/' filter=lfs diff=lfs merge=lfs -text
'*.html' filter=lfs diff=lfs merge=lfs -text
'*.svg' filter=lfs diff=lfs merge=lfs -text
```
5. I then add a `test.svg` file to `myfolder/` and `git add "myfolder/test.svg"`.
6. To test whether the file is, in fact, being sent to LFS, I run `git lfs ls-files` which returns nothing. `git lfs status` returns:
```
On branch main
Objects to be pushed to origin/main:


Objects to be committed:

        myfolder/test.svg (Git: 022e3d9)

Objects not staged for commit:

...
```
The wording here is also not "**Git LFS** objects to be..." etc. as mentioned [here](https://stackoverflow.com/questions/54451856/how-can-i-tell-if-a-file-will-be-uploaded-to-git-lfs-correctly) and the parentheses behind the file name also do not indicate `(LFS: [hash])` as in the same link.
7. After committing and pushing test.svg, I also do not see the file indicated as being hosted on Git LFS when viewing it on GitHub, so clearly I don't have LFS set up correctly.

Would you be able to provide any pointers to what I'm doing wrong?

**Comment by chrisd8088** [comment]
Hey, sorry you're having trouble.  How large is your `test.svg` file, and is it empty, by chance?  That makes a difference, for me:
```
$ git init test
$ cd test
$ git lfs track '*.svg'
$ touch empty.svg
$ echo test >test.svg
$ git add *.svg
$ git lfs status

Objects to be committed:

	empty.svg (Git: e3b0c44)
	test.svg (LFS: f2ca1bb)

Objects not staged for commit:


$ git commit -m svg
$ git lfs ls-files
f2ca1bb6c7 * test.svg
```

**Reply by gregorhd** [reply]
Hi Chris! Thanks a mil. The .svg is about 2.5MB and [not empty](https://github.com/gregorhuh/mapcompare/blob/main/mapcompare/outputs/static/geoviews%20(dd_subset).svg). I have other files which are beyond the regular upload limit for GitHub (40-240MB). I tested LFS with these files before but they ended up causing a major hassle once committed, then not picked up by LFS and causing me to hard reset to earlier commits, so I'm testing the mechanism with smaller files now.

Would `git add --renormalize` do anything as hinted at [here](https://github.com/git-lfs/git-lfs/issues/4070)? I don't exactly understand what it does and when to run it (after `git add test.svg`?).

**Reply by gregorhd** [reply]
Does anybody have any ideas?
